# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/CacheIRCompiler.h
# Commit: ae6126131b6d
# Full Hash: ae6126131b6dfa18981f25c6986200c14e63e5f2
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2024-04-11 04:26:26
# Regressor Bug: 1881995
# File Overlap Count: 1
# Description:
#   Bug 1881995 - Implement ForwardedAtoms and create them during atomization r=iain,sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D202690
# ==============================================================================

diff -r 4e5967329fdd -r ae6126131b6d js/src/jit/CacheIRCompiler.h
--- a/js/src/jit/CacheIRCompiler.h	Wed Apr 10 15:47:15 2024 +0000
+++ b/js/src/jit/CacheIRCompiler.h	Wed Apr 10 16:32:23 2024 +0000
@@ -1420,6 +1420,9 @@
 
   void replaceStubRawWord(uint8_t* stubData, uint32_t offset, uintptr_t oldWord,
                           uintptr_t newWord) const;
+
+  void replaceStubRawValueBits(uint8_t* stubData, uint32_t offset,
+                               uint64_t oldBits, uint64_t newBits) const;
 };
 
 template <typename T>